etesync_server/settings.py

```diff
@@ -23,7 +23,7 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 # See secret.py for how this is generated; uses a file 'secret.txt' in the root
 # directory
-SECRET_KEY = get_secret_from_file(os.path.join(BASE_DIR, "secret.txt"))
+SECRET_FILE = os.path.join(BASE_DIR, "secret.txt")
```

Comment:
I'd just move this, as it was, to line 147. No need to have it split.

Reply:
The thought was that I'd just set a new `SECRET_FILE` setting rather than having to import and do the `get_secret_from_file` step manually.

Reply:
Fair point. No need to change then.